src/mpid/ch4/shm/posix/posix_init.h

```diff
@@ -105,11 +107,14 @@ static inline int MPIDI_POSIX_mpi_init_hook(int rank, int size, int *n_vnis_prov
         MPIDI_POSIX_global.active_rreq[i] = NULL;
     }
 
+    /* XXX - Is the eager submodule never inlined? We might want to do that for certain situations,
+     * for instance, if only one eager submodule is compiled in. */
```

Comment:
Is the eager submodule never inlined? We might want to do that for certain situations. For instance, if only one eager submodule is compiled in.